frameworks/AutoGluon/setup.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+HERE=$(dirname "$0")
+
+# creating local venv
```

Comment:
Regarding libomp, this is the setup code I use for internal testing:

```
if [[ "$OSTYPE" == "darwin"* ]]; then
    echo "Operating System is Mac, running homebrew install of libomp to enable LightGBM"
    # Install libomp to support LightGBM package on mac
    if brew ls --versions libomp > /dev/null; then
        echo "libomp already installed..."
    else
        brew install libomp
    fi
fi
```

I don't believe libomp installation is required on other operating systems, and I'm not sure how `SUDO apt-get install -y libomp-dev` differs from `brew install libomp`. I'm curious to know if there is a difference, since I would be interested in adding it to our setup documentation to assist users that don't have brew installed.

Reply:
I'm just making the assumption that macOS users will have brew installed (others have to install missing dependencies manually).
The `apt-get install` is mainly for debian-based distros: this is what we're using to build the docker images, and/or to run the benchmarks on AWS.

Reply:
You are saying that debian-based distros don't work out of the box with LightGBM? I have only tested with Ubuntu which does not require any installation. If so I will add that to AutoGluon's [installation instructions](https://autogluon.mxnet.io/).

Reply:
to be honest, I didn't try without the current state of this `setup.sh` file.
I'm working on macOS and as I had to install `libomp` there, I made the assumption that it was also needed on Ubuntu, but if you're saying that it's not necessary, then I'll try to remove it.